redfish-core/lib/memory.hpp

```diff
@@ -460,10 +458,6 @@ inline void getDimmDataByService(std::shared_ptr<bmcweb::AsyncResp> aResp,
                                  const std::string& service,
                                  const std::string& objPath)
 {
-    auto health = std::make_shared<HealthPopulate>(aResp);
```

Comment:
HealthRollup is not required for the DIMM in the OpenPOWER platforms like upstream?